src/utils/generate-vdm-util.ts

```diff
@@ -0,0 +1,49 @@
+/*!
+ * Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved.
+ */
+
+import { flags } from '@oclif/command';
+import { AlphabetLowercase, AlphabetUppercase } from '@oclif/parser/lib/alphabet';
+import { IBooleanFlag, IOptionFlag } from '@oclif/parser/lib/flags';
+import { generatorOptionsCli as generatorOptionsSDK, GeneratorOptions } from '@sap/cloud-sdk-generator';
+import * as path from 'path';
+import { Options } from 'yargs';
+import { FlagsParsed } from '../commands/generate-vdm';
+
+export function toGeneratorSDK(flags: FlagsParsed): GeneratorOptions {
+  const result =  Object.entries(flags)
```

Comment:
The flags parameter shadows the import from '@oclif/command'. Changing the parameter name to `cliFlags` or something like that could be good.

Reply:
uh, dangerous.